### PR TITLE
Fixed Home Tab Mobile View UI

### PR DESCRIPTION
**Pull request by CurtisUnderwood**
### Description

List of proposed changes:

- Updated the mobile UI for the Home page tab system so that the designs are more consistent with the desktop version
- Tab text is now dynamic with what tab is being displayed

### Additional Notes

### CODE REVIEW

File: frontend/src/pages/home.tsx [hunk 1]
```diff
@@ -113,19 +113,22 @@ const Home: FC = () => {
                 </TabList>
               )}
               {extraSmall && (
-                <div className="px-4 py-2">
+                <div className="">
```
Comment (sebastien-comeau) [inline frontend/src/pages/home.tsx:116]: Use React.Fragment `<></>` instead of Div

File: frontend/src/pages/home.tsx [hunk 2]
```diff
@@ -113,19 +113,22 @@ const Home: FC = () => {
...
                   <div className="flex gap-1">
```
Comment (sebastien-comeau) [inline frontend/src/pages/home.tsx:117]:
```suggestion
                  <div className="grid grid-cols-2 gap-1">
```

File: frontend/src/pages/home.tsx [hunk 3]
```diff
@@ -113,19 +113,22 @@ const Home: FC = () => {
...
+                    <div className='border-b-2 w-1/2 text-center border-primary-700 p-2'>
```
Comment (sebastien-comeau) [inline frontend/src/pages/home.tsx:118]:
```suggestion
                    <div className="border-b-2 border-primary-700 p-2">
```

File: frontend/src/pages/home.tsx [hunk 4]
```diff
@@ -113,19 +113,22 @@ const Home: FC = () => {
...
+                      <Button
+                        variant="text"
+                        onClick={() => setValue('learn')}
+                        className="grow text-lg font-bold"
+                      >
+                        {t(`tabs.${value}.title`)}
+                      </Button>
```
Comment (sebastien-comeau) [inline frontend/src/pages/home.tsx:125]:
```suggestion
                      <Button
                        variant="text"
                        onClick={() => setValue('learn')}
                        className="grow text-lg font-bold"
                        fullWidth
                      >
                        {t(`tabs.${value}.title`)}
                      </Button>
```

File: frontend/src/pages/home.tsx [hunk 5]
```diff
@@ -113,19 +113,22 @@ const Home: FC = () => {
...
-                      className="grow underline underline-offset-4"
-                    >
-                      {t('tabs.learn.title')}
-                    </Button>
-                    <Button
+                      variant="text"
                       onClick={() => setOpen(!open)}
                       endIcon={open ? <ExpandLess /> : <ExpandMore />}
-                      className="grow"
+                      className="grow text-lg font-bold text-[#666666]"
```
Comment (sebastien-comeau) [inline frontend/src/pages/home.tsx:131]:
```suggestion
                      className="text-lg font-bold text-black text-opacity-60"
                      fullWidth
```

**Review by sebastien-comeau — COMMENTED** [review]
Some change and please remove `,` after `Tab` component:

image

File: frontend/src/pages/home.tsx [hunk 6]
```diff
@@ -113,19 +113,22 @@ const Home: FC = () => {
                 </TabList>
               )}
               {extraSmall && (
-                <div className="px-4 py-2">
+                <div className="">
                   <div className="flex gap-1">
+                    <div className='border-b-2 w-1/2 text-center border-primary-700 p-2'>
+                      <Button
+                        variant="text"
+                        onClick={() => setValue('learn')}
```
Comment (sebastien-comeau) [inline frontend/src/pages/home.tsx:121]: Is it normal that onClick you set the current tabl to `learn` even if the text is different?

Comment (CurtisUnderwood) [reply frontend/src/pages/home.tsx]: No, not sure why it's there to begin with I removed it

File: frontend/src/pages/home.tsx (hunk 5 above)
Comment (CurtisUnderwood) [reply frontend/src/pages/home.tsx]: Fixed

File: frontend/src/pages/home.tsx (hunk 4 above)
Comment (CurtisUnderwood) [reply frontend/src/pages/home.tsx]: Fixed

File: frontend/src/pages/home.tsx (hunk 3 above)
Comment (CurtisUnderwood) [reply frontend/src/pages/home.tsx]: Fixed

File: frontend/src/pages/home.tsx (hunk 2 above)
Comment (CurtisUnderwood) [reply frontend/src/pages/home.tsx]: Fixed

File: frontend/src/pages/home.tsx (hunk 1 above)
Comment (CurtisUnderwood) [reply frontend/src/pages/home.tsx]: Fixed

File: frontend/src/pages/home.tsx [hunk 7]
```diff
@@ -107,25 +107,29 @@ const Home: FC = () => {
...
+                      >
+                        {t(`tabs.${value}.title`)}
+                      </Button>
+                    </div>
                     <Button
+                      variant="text"
                       onClick={() => setOpen(!open)}
                       endIcon={open ? <ExpandLess /> : <ExpandMore />}
-                      className="grow"
+                      className="grow text-lg font-bold text-black text-opacity-60"
```
Comment (sebastien-comeau) [inline frontend/src/pages/home.tsx:131]: Remove `grow` class.

Comment (CurtisUnderwood) [reply frontend/src/pages/home.tsx]: Fixed

**Review by sebastien-comeau — APPROVED** [review]

**Pull request merged**